### PR TITLE
Update CN README & guomi keycenter & log format

### PR DESCRIPTION
**Pull request by JimmyShi22**
(1) Update Chinese README
(2) Add guomi support for keycenter
(3) Update log format of sync

### CODE REVIEW

File: libdevcore/BasicLevelDB.cpp [hunk 1]
```diff
@@ -49,14 +49,15 @@ void LevelDBWriteBatch::kill(Slice _key)
...
+  : m_name(_name)
 {
     // Basic leveldb initralization(No encryption)
     auto db = static_cast<leveldb::DB*>(nullptr);
     m_openStatus = leveldb::DB::Open(_options, _name, &db);
 
     if (!m_openStatus.ok() || !db)
     {
-        LOG(ERROR) << "Database open error" << endl;
+        cerr << "Database open error" << endl;
```
Comment (bxq2011hust) [inline libdevcore/BasicLevelDB.cpp]: what about LOG also output this error?

File: libsecurity/EncryptedLevelDB.cpp [hunk 2]
```diff
@@ -213,14 +231,14 @@ leveldb::Status EncryptedLevelDB::Put(
...
-        //                << "/" << ascii2hex(_value.data(), _value.size()) << endl;
+        // ENCDB_LOG(TRACE)<< LOG_BADGE("ENC")<< LOG_DESC("Put")<<
+        // LOG_KV("k",ascii2hex(_key.data(), _key.size()))<< LOG_KV("encv",ascii2hex(enData))<<
+        // LOG_KV("v", ascii2hex(_value.data(), _value.size()));
     }
     catch (Exception& e)
     {
-        ENCDBLOG(ERROR) << "[Put] Encrypt ERROR! [k/v]: " << _key.ToString() << "/ "
-                        << _value.ToString() << endl;
+        ENCDB_LOG(ERROR) << "[Put] Encrypt ERROR! [k/v]: " << _key.ToString() << "/ "
```
Comment (cyjseagull) [inline libsecurity/EncryptedLevelDB.cpp]: This log hasn't  been modified.

File: libsecurity/EncryptedLevelDB.h [hunk 3]
```diff
@@ -34,16 +34,21 @@ namespace dev
 {
 namespace db
 {
-#define ENCDBLOG(_OBV) LOG(_OBV) << " [ENCDB] "
+#define ENCDB_LOG(_OBV)                 \
+    LOG(_OBV) << "[g:" << m_name << "]" \
```
Comment (cyjseagull) [inline libsecurity/EncryptedLevelDB.h]: refer to use m_group~

File: libdevcore/BasicLevelDB.cpp (hunk 1 above)
Comment (JimmyShi22) [reply libdevcore/BasicLevelDB.cpp]: done

File: libsecurity/EncryptedLevelDB.cpp (hunk 2 above)
Comment (JimmyShi22) [reply libsecurity/EncryptedLevelDB.cpp]: Yes! Done

File: libsecurity/EncryptedLevelDB.h (hunk 3 above)
Comment (JimmyShi22) [reply libsecurity/EncryptedLevelDB.h]: Waiting for group log plan

**Pull request merged**